docs/sql/statements/explain.rst

```diff
@@ -27,8 +28,11 @@ The ``EXPLAIN`` command displays the execution plan that the planner generates
 for the supplied statement. The plan is returned as a nested object containing
 the plan tree.
 
-When issuing ``EXPLAIN ANALYZE`` the plan of the statement is executed and
-timings of the different phases of the plan are returned.
+When issuing ``EXPLAIN ANALYZE`` or ``EXPLAIN (ANALYZE TRUE)`` the plan of the
+statement is executed and timings of the different phases of the plan are returned.
+
+When issuing ``EXPLAIN (COSTS FALSE)`` the execution plan does not contain the
+costs for each logical plan in the execution plan.
```

Comment:
We could mention that `costs true` is the default and it will print the estimated rows.

Reply:
👍